packages/inferno-router/src/BrowserRouter.ts

```diff
@@ -15,15 +16,16 @@ export interface IBrowserRouterProps {
 export class BrowserRouter extends Component<IBrowserRouterProps, any> {
   public history;
 
-  constructor(props?: any, context?: any) {
+  constructor(props?: IBrowserRouterProps, context?: any) {
     super(props, context);
-    this.history = createBrowserHistory(props);
+    this.history = createBrowserHistory(/*props*/); // TODO: None of the props are defined in createBrowserHistory so skipping
```

Comment:
What props the method have? Shouldnt we fix typing rather than change behavior?

Reply:
The only property createBrowserHistory accepts is `window`, but we don't accept it as a prop to BrowserRouter. createBrowserHistory defaults `window = document.defaultView! ` so passing props here is a NOOP. Apparently one can pass another window object if one likes but I am not sure what use that is and what the side effects are.

We have two options:
1. remove props as above
2. add `window?: Window` to IBrowserRouterProps

What do you think makes more sense? NOTE: I didn't research this at all because it felt like out of scope, I just fixed the type error since it was a NOOP and assumed it was a copy/paste error.

Ref: https://github.com/remix-run/history/blob/main/packages/history/index.ts

Reply:
Yeah based on your comments removing it makes sense. I can check it as well once I'm home.

Reply:
@Havunen did you reach some conclusion on this? If we remove it, it will be easy enough to reintroduce if someone raises an issue.

Reply:
yeah lets remove it